openid-federation-1_0.xml

```diff
@@ -3523,70 +3523,64 @@
         </section>
 
         <section title="Validating a Trust Mark" anchor="trust_valid">
+          <t>
+            Validating a Trust Mark means validating a Trust mark instance
+            represented by a specific signed JWT.
+            It is NOT about validating whether a Trust Mark of a special kind
+            can exist or not.
+          </t>
           <t>
 	    An Entity SHOULD NOT try to validate a Trust Mark until
 	    it knows which Trust Anchor it is using.
 	    To validate a Trust Mark Issuer, follow the procedure
 	    defined in <xref target="resolving_trust"/>.
-	    To validate a Trust Mark, either:
-            <list style="symbols">
-              <t>
-                Validate the signature of the Trust Mark JWT and verify that
-                it has not expired.
-              </t>
-              <t>
-                If the Trust Mark type identifier appears in the
-                <spanx style="verb">trust_mark_owners</spanx> claim
-		of the Trust Anchor's Entity Configuration,
-                verify that the Trust Mark contains
-                a <spanx style="verb">delegation</spanx>
-                claim.
-                The claims for the Trust Mark type identifier
-                in the <spanx style="verb">trust_mark_owners</spanx> value
-                are used in the following way:
-                <list style="symbols">
-                  <t>
-                    The signature of the <spanx style="verb">delegation</spanx> JWT
-                    MUST verify with a key from <spanx style="verb">jwks</spanx> claim.
-                  </t>
-                  <t>
-                    The issuer of the <spanx style="verb">delegation</spanx> JWT
-		    MUST match the <spanx style="verb">sub</spanx> value
-		    in this set of claims.
-                  </t>
-                </list>
-              </t>
-	    </list>
-	  or:
-            <list style="symbols">
-              <t>
-                Use the Trust Mark Issuer status endpoint to verify that the
-                Trust Mark is still active
-                as described in <xref target="status_endpoint"/>.
+          </t>
+      <t>
+        Henceforth instance is used as a shorthand for trust mark instance. An
+        instance is a signed JWT.
+        To validate an instance the following has to be done (if any of these
+        tests fail then the validation fails):
+        <list style="numbers">
+          <t>
+            Verify that the value of the instance’s
+            <spanx style="verb">trust_mark_id</spanx>  is listed in
```

Comment:
Hint for trust_mark_id > trust_mark_type change: https://github.com/openid/federation/pull/219

Reply:
does this have implications in line 3551 with the id should be 'type'?